docs/api/dt/cummin.rst

```diff
@@ -0,0 +1,87 @@
+
+.. xfunction:: datatable.cummin
+    :src: src/core/expr/fexpr_cumminmax.cc pyfn_cummin
+    :tests: tests/dt/test-cumminmax.py
+    :cvar: doc_dt_cummin
+    :signature: cummin(cols)
+
+    .. x-version-added:: 1.1.0
+
+    For each column from `cols` calculate cumulative minimum. In the presence of :func:`by()`,
+    the cumulative minimum is computed per group.
+
+    Parameters
+    ----------
+    cols: FExpr
+        Input data for cumulative minimum calculation.
+
+    return: FExpr
+        f-expression that converts input columns into the columns filled
+        with the respective cumulative minimums.
+
+    except: TypeError
+        The exception is raised when one of the columns from `cols`
+        has a non-numeric type.
+
+
+    Examples
```

Comment:
@samukweku May I ask you to adjust your examples for `cummin()`? Currently I just copy-pasted these examples from `cummax()`.